WordPress/src/main/java/org/wordpress/android/ui/sitecreation/progress/SiteCreationProgressViewModel.kt

```diff
@@ -1,4 +1,4 @@
-package org.wordpress.android.ui.sitecreation.progress
+﻿package org.wordpress.android.ui.sitecreation.progress
```

Comment:
I don't see what is different here :thinking:

Reply:
Strange! Might be because it doesn't place the whitespace change on the right line of code, it seems to come from commit 6717a387 according to the AS blame